dbschema/models.py

```diff
@@ -158,7 +160,7 @@ class UserInterventionState(Base):
     intervention_phase_id = Column(Integer, ForeignKey('intervention_phases.phase_id'))
     intervention_component_id = Column(Integer, ForeignKey('intervention_components.intervention_component_id'))
     completed = Column(Boolean)
-    last_time = Column(TIMESTAMP(timezone=True), default=func.now())
+    last_time = Column(TIMESTAMP(timezone=True), default=func.now(), nullable=True)
```

Comment:
Actually I really do not remember exactly why I changed this. There was no bug, but probably it makes sense to have a null value in case the user has not done anything in the dialog